WinUSBNet/API/WinUSBDevice.cs

```diff
@@ -138,7 +138,7 @@ public string GetStringDescriptor(byte index, ushort languageID)
             return new string(chars);
         }
 
-        public void ControlTransfer(byte requestType, byte request, ushort value, ushort index, ushort length, byte[] data)
+        public uint ControlTransfer(byte requestType, byte request, ushort value, ushort index, ushort length, byte[] data)

```

Comment:
Maybe let this return int as well for consistency?

Reply:
Yeah, good idea. Will do.

Reply:
Done; please check the changes below.